trieste/models/interfaces.py

```diff
@@ -97,7 +97,7 @@ def get_kernel(self) -> gpflow.kernels.Kernel:
         Return the kernel of the model.
         :return: The kernel.
         """
-        raise NotImplementedError(f"Model {self!r} does not provide observation noise")
+        raise NotImplementedError(f"Model {self!r} does not provide a kernel")
```

Comment:
should probably go into a different PR, we don't know when we will merge this one...

Reply:
there was one f string above as well, perhaps a quick PR with these small fixes?

Reply:
Done, see #336